src/content/staking/withdrawals/index.md

```diff
@@ -143,7 +143,7 @@ As an alternative to changing the withdrawal address for a particular validator,
 <p>Once a validator has completed the exiting process, and assuming the account has withdrawal credentials, the remaining balance will <em>then</em> be withdrawn during the next validator sweep.</p>
 </ExpandableCard>
 
-<ExpandableCard title="How can I withdrawal a custom amount?">
+<ExpandableCard title="How can I withdraw a custom amount?">
```

Comment:
@wackerow I think the current open wording implies that custom amount withdrawals are possible. I'd personally phrase this as a closed question. WDUT?

```suggestion
<ExpandableCard title="Can I withdraw a custom amount?">
```

Reply:
I agree this wording is better